docs/source/getting_started/running_at_scale.ipynb

```diff
@@ -23,12 +23,7 @@
     "\n",
```

Comment:
<p>Should this be a separate document (with a Next button from the Register page). Might be easier to maintain long term.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Potentially yes... although it's kind of maintained separately because it is a reference to the Jobs UI documentation. Maybe an example-specific walkthrough on a new Getting Started page, plus the link to the Jobs UI doc.</p>

Reply:
<p>will address in future update. git issue: https://github.com/MAAP-Project/maap-documentation/issues/277</p>

Reply:
how about links to the more detailed docs?

Reply:
Beyond the links to the Jobs UI, submitting via Jobs UI and Monitoring via Jobs UI?

Reply:
Ah I missed those.